app/views/records/edit_fields/_rights_statement.html.erb

```diff
@@ -0,0 +1,5 @@
+<% rights_statements = Hyrax::RightsStatementService.new %>
+<%= f.input :rights_statement,
+    collection: rights_statements.select_active_options,
+    include_blank: true,
+    input_html: { class: 'form-control', name: "#{curation_concern.class.to_s.downcase}[rights_statement][]" } %>
```

Comment:
For this string interpolation:
`"#{curation_concern.class.to_s.downcase}[rights_statement][]"`
What happens if the `curation_concern.class` is `GenericObject`?  I think you probably need to use the `underscore` method instead of `downcase`

Reply:
Could you add a a rights statement to one of the specs, such as `spec/features/create_generic_object_spec.rb` and make sure that the name coverts properly and the work is saved with the correct rights statement?  Let me know if you need any help wrangling the capybara specs.